src/common/modules/UnicodeTransformationHandler.js

```diff
@@ -53,6 +53,7 @@ export function getTransformationType(transformationId) {
 function capitalizeEachWord(text) {
     // Regular expression Unicode property escapes and lookbehind assertions require Firefox/Thunderbird 78
     // see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp#bcd:javascript.builtins.RegExp
+    // \p{Alphabetic}
```

Comment:
Was this intended? If so, what should this comment mean?

Especially since the actual RegEx uses `\P{Alpha}`, which is different to what you write above.

Reply:
`\p{Alpha}` is short for `\p{Alphabetic}`. Most of the Unicode property escapes have shorter versions/aliases for use in production (see [here](https://tc39.es/ecma262/#table-binary-unicode-properties)). I thought it would still be good to document the longer version for reference.

Reply:
> have shorter versions/aliases for use in production

Uhm, and just using the larger version directly in the RegEx would not work or is less compatible?

As we’d maybe want self explaining/clean code.

Reply:
> Uhm, and just using the larger version directly in the RegEx would not work or is less compatible?

The longer version should work. There is very little documentation on the Unicode property escapes feature since it is so new, but I am guessing that the shorter versions are faster for the browser to parse, compile and optimize, which is why they are provided. Considering that these have to be run every time the user opens the context menu with the live preview option, I wanted them to be as fast as possible so they do not experience a blink.

Reply:
>  I am guessing that the shorter versions are faster for the browser to parse, compile and optimize, which is why they are provided

As a “saying” says: without performance benchmarks, you cannot say where performance goes and it can get easily faster or slower. (Note there are JS performance websites that just execute a code x times, if you wanna really try it though.)

Personally, I would doubt it changes a lot, given the length of the string hardly matters, as it’s one atomic unit the browser parses, so well… dunno…, but I could be wrong of course.

Reply:
> As a “saying” says: without performance benchmarks, you cannot say where performance goes and it can get easily faster or slower. (Note there are JS performance websites that just execute a code x times, if you wanna really try it though.)

True, although benchmarking such a small snippet of JS would be difficult and likely very noisy.

> Personally, I would doubt it changes a lot, given the length of the string hardly matters, as it’s one atomic unit the browser parses, so well… dunno…, but I could be wrong of course.

Yeah, it is the only reason I could think of that the shorter versions were even provided, otherwise everyone would just use the longer versions, although I could be wrong too. Anyway, I am happy to change these regular expressions to use the longer versions of the Unicode property escapes if you want.

Reply:
> Yeah, it is the only reason I could think of that the shorter versions were even provided,

People and standards very often do strange things , so… :stuck_out_tongue_winking_eye: 

And if that were the reason, the standard _would_ (or at least should) document/describe that as the reason.

> Anyway, I am happy to change these regular expressions to use the longer versions of the Unicode property escapes if you want.

Well… I don’t know as I’m also not sure, as you see. If your arguing is correct I would definitively prefer your version for performance reasons, otherwise mine for readability reasons. :shrug: 
So unless we find a resource or so and if both things are compatible, I’m fine with leaving it as it is.

Reply:
> And if that were the reason, the standard _would_ (or at least should) document/describe that as the reason.

I read [the standard](https://tc39.es/ecma262/#sec-runtime-semantics-unicodematchproperty-p) as well as the [original proposal](https://github.com/tc39/proposal-regexp-unicode-property-escapes) and there is no reason for the “aliases” given. 😕

> So unless we find a resource or so and if both things are compatible, I’m fine with leaving it as it is.

OK, I guess I will leave it as is… I tried to have both performance and readability by adding these comments.